docs/src/content/docs/reference/helm-chart-config.mdx

```diff
@@ -0,0 +1,749 @@
+                `fieldName_A`, `fieldName_B` in the case of an organism with segments `A` and `B`.
+            </td>
+        </tr>
+        <tr>
+            <td>`customDisplay`</td>
+            <td>Object</td>
+            <td></td>
+            <td>Custom display of the field on the sequence details page</td>
+        </tr>
+        <tr>
+            <td>`customDisplay.type`</td>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`customDisplay.url`</td>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`rangeSearch`</td>
+            <td>Boolean</td>
+            <td></td>
+            <td>If true, enables range search for numeric fields.</td>
+        </tr>
+        <tr>
+            <td>`ontology_id`</td>
+            <td>String</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`definition`</td>
+            <td>String</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`guidance`</td>
+            <td>String</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`example`</td>
+            <td>String</td>
+            <td></td>
+            <td>Example value for the field.</td>
+        </tr>
+        <tr>
+            <td>`ingest`</td>
+            <td>String</td>
+            <td></td>
+            <td>Which NCBI field to map to this field</td>
+        </tr>
+        <tr>
+            <td>`preprocessing`</td>
+            <td>[PreprocessingFunction (type)](#preprocessingfunction-type)</td>
+            <td>Identity function on input field with same name</td>
+            <td>Preprocessing function to run on the field</td>
+        </tr>
+    </tbody>
+</table>
+
+### PreprocessingFunction (type)
+
+### Preprocessing (type)
+
+<table>
+    <thead>
+        <tr>
+            <th className='w-48'>Field</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`version`</td>
+            <td>Integer</td>
+            <td></td>
+            <td>Version of the preprocessing pipeline</td>
+        </tr>
+        <tr>
+            <td>`image`</td>
+            <td>String</td>
+            <td></td>
+            <td>Docker image for the preprocessing pipeline</td>
+        </tr>
+        <tr>
+            <td>`args`</td>
+            <td>Array of strings</td>
+            <td></td>
+            <td>Arguments passed to the preprocessing pipeline</td>
+        </tr>
+        <tr>
+            <td>`configFile`</td>
+            <td>Object</td>
+            <td></td>
+            <td>Configuration file for the preprocessing pipeline</td>
+        </tr>
+    </tbody>
+</table>
+
+### Ingest (type)
```

Comment:
<table>
    <thead>
        <tr>
            <th className='w-48'>Field</th>
            <th>Type</th>
            <th>Default</th>
            <th>Description</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>`image`</td>
            <td>String</td>
            <td></td>
            <td>Docker image for the preprocessing pipeline</td>
        </tr>
        <tr>
            <td>`configFile`</td>
            <td>Object</td>
            <td></td>
            <td>Configuration file for the ingest pipeline</td>
        </tr>
    </tbody>
</table>

The config file needs to contain the taxon_id of the organism and additionally, if the organism is multi-segmented, it requires a list of segment names (`nucleotide_sequences`) and the `nextclade_dataset` that can be used for segment identification and alignment.

Reply:
Thanks, I added it and also added a link to the folder of the ingest pipeline on GitHub.